src/structures/ClientApplication.js

```diff
@@ -23,6 +30,33 @@ class ClientApplication extends Application {
   _patch(data) {
     super._patch(data);
 
+    if ('tags' in data) {
+      /**
+       * The tags this application has
+       * @type {?string[]}
+       */
+      this.tags = data.tags;
+    }
+
+    if ('install_params' in data) {
+      /**
+       * Settings for this application's default in-app authorization
+       * @type {?ClientApplicationInstallParams}
+       */
+      this.installParams = {
+        ...data.install_params,
```

Comment:
This sounds dangerous, what if they add a new property and that property is snake cased?